test/examples_test.go

```diff
@@ -0,0 +1,648 @@
+package test
```

Comment:
This file is really big. I recommend to split it by module to be tested.

Reply:
I would suggest splitting it by example to be tested and maybe a file with helper functions.

Reply:
I agree to the suggested splitting. Having a helper file and some test files is a good approach.

Reply:
Fixed with 1b78704.